scss/templates/user.scss

```diff
@@ -198,3 +148,188 @@
     }
   }
 }
+
+.stats-section {
+  .stats-title {
+    display: none;
+  }
+
+  ul {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  li,
+  .linked-stat {
+    margin: 0;
+    padding: 0;
+    margin: 0 1.5rem 1.5rem 0;
+
+    > a {
+      padding: 0 !important;
+    }
+  }
+
+  .user-stat {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 10rem;
+    height: 7.5rem;
+    background-color: $blue;
+    border-radius: 0.25rem;
+
+    .number,
+    .label,
+    .value {
+      color: $white;
+      font-style: normal;
+      font-weight: normal;
+    }
+
+    .number,
+    .value {
+      font-size: 2.25rem;
+      font-family: "Roboto", $font-family-sans-serif;
```

Comment:
You really need this? I wonder:

1. Do we have a different font family for a set of things within the admin? If so maybe is a mistake
2. Can we use variables that we already define with the font family?
3. Those this text can benefit from something like `@extend .dc-text-monospace` _(take a look to `type.scss`)_

Reply:
I took this from the design

![image](https://user-images.githubusercontent.com/20747535/89696583-2eef5e80-d8de-11ea-8e98-09d86ec31c21.png)

I could use monospace 💯

Reply:
Monospace looks very different from the designs

![image](https://user-images.githubusercontent.com/20747535/89696621-55ad9500-d8de-11ea-9e0e-b4d38b07cfa0.png)

I think for this case we could still be using `Libre Franklin`

Reply:
This is how it looks with Libre Franklin 

![image](https://user-images.githubusercontent.com/20747535/89696658-855c9d00-d8de-11ea-8d62-cccb712fe9b0.png)

Reply:
I guess we can use the Libre Frankling font here, looks good to me.